app/components/building-blocks/Price.tsx

```diff
@@ -0,0 +1,20 @@
+import { currency, useTicket } from '@/app/TicketContext'
+
+export default function Price() {
+    const { data } = useTicket()
+
+    const formatCurrency = (value: number, curr: currency) => {
+        switch (curr) {
+            case currency.SEK:
+                return value.toFixed(0)
+            default:
+                return value.toFixed(2)
+        }
```

Comment:
I'd rather use a useMemo hook to calculate the memory. Otherwise the function is built and called on each render of the component

Reply:
Sure :) I assume this is a performance improvement and not a functional thing?